docs/src/index.md

```diff
@@ -210,7 +210,7 @@ Numerical Results of Distributed Computations
 
 Floating point arithmetic is not associative and this comes up
 when performing distributed computations over `DArray`s.  All `DArray`
-operations are performed over the `localpart` chunks and then aggregated.
+operations are performed over the localparts and then aggregated.
```

Comment:
I'm not sure this is an improvement. It might be useful to emphasize that the operations are performed over what `localpart` returns.

Reply:
Indeed. And thanks for the contribution.